src/sdk/pynni/nni/compression/torch/builtin_quantizers.py

```diff
@@ -227,20 +227,17 @@ class DoReFaQuantizer(Quantizer):
     (https://arxiv.org/abs/1606.06160)
     """
     def __init__(self, model, config_list):
```

Comment:
better to add docstring to explain `model` and `config_list`

Reply:
fixed